src/duke/main/Duke.java

```diff
@@ -0,0 +1,223 @@
+package duke.main;
+
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.Event;
+import duke.task.Todo;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke{
+    public static final String LINEBREAK = "____________________________________________________________";
+    public static final String FILEPATH = "data/duke.txt";
+    public static final String DIRPATH = "data";
+    public static ArrayList<Task> list = new ArrayList<>();
+    public static Scanner in = new Scanner(System.in);
+    public static int taskCount = 0;
+    public static boolean isLoaded = false;
+    public static void loadSavedTasks(){
+        try{
+            File dirPath = new File(DIRPATH);
+            if (!dirPath.exists()){
+                dirPath.mkdirs();
+            }
+            File file = new File(FILEPATH);
+            Scanner txtScanner = new Scanner(file);
+            while (txtScanner.hasNext()) {
+                isLoaded = true;
+                String line = txtScanner.nextLine();
+                String[] fields = line.split(" \\| ");
+                switch(fields[0]){
```

Comment:
Avoid using magic number such as `0`. Consider extracting them into a constant, or alternatively you can separate the array elements to create new attributes to improve code readability.